src/main/java/com/linkedin/kmf/services/MultiClusterTopicManagementService.java

```diff
@@ -262,7 +262,9 @@ public void run() {
       Map topicFactoryConfig = props.containsKey(TopicManagementServiceConfig.TOPIC_FACTORY_PROPS_CONFIG) ?
           (Map) props.get(TopicManagementServiceConfig.TOPIC_FACTORY_PROPS_CONFIG) : new HashMap();
       _topicFactory = (TopicFactory) Class.forName(topicFactoryClassName).getConstructor(Map.class).newInstance(topicFactoryConfig);
+
       _adminClient = constructAdminClient(props);
+      LOG.info("{} configs: {}", _adminClient.getClass().getSimpleName(), _adminClient);
```

Comment:
The second argument to the logger is of type `AdminClient`. Do you mean to use `props` instead of `_adminClient `?